packages/web_benchmarks/README.md

```diff
@@ -0,0 +1,15 @@
+# web_benchmarks
+
+A benchmark harness for Flutter Web apps. Currently only supports running
+benchmarks in Chrome.
+
+# Writing a benchmark
+
+An example benchmark can be found in [test/web_benchmark_test.dart][1].
+
+A web benchmark is made of two parts: a client and a server. The client is code
+that runs in the browser together with the benchmark code. The server serves the
+app's code and assets. Additionally, the server communicates with the browser to
+extract the performance traces.
+
+[1]: https://github.com/flutter/packages/tree/packages/web_benchmarks/test/web_benchmarks_test.dart
```

Comment:
This link doesn't work btw. There has to be a branch or sha after `/tree/`.

Reply:
It's because it's pointing at itself. It will work once the PR is submitted.

Reply:
I know this is a new file so it won't work. But I tried with an existing file, still didn't work. You have to tell it which tree to look at :)